tools/topology/platform/common/ssp.m4

```diff
@@ -39,9 +39,14 @@ define(`SSP_CONFIG_DATA',
 `		SOF_TKN_INTEL_SSP_SAMPLE_BITS'	STR($3)
 `		SOF_TKN_INTEL_SSP_QUIRKS'	`ifelse(`$5', `', "0", STR($5))'
 `		SOF_TKN_INTEL_SSP_BCLK_DELAY'	`ifelse(`$6', `', "0", STR($6))'
+`		SOF_TKN_INTEL_SSP_CLKS_CONTROL' `ifelse(`$7', `', "0", STR($7))'
```

Comment:
missed updating the macro definition comment above?

Reply:
ok, added